x-pack/plugins/siem/public/components/stat_items/areachart.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+// @ts-ignore
+import { EuiSeriesChart, EuiAreaSeries, EuiXAxis, EuiYAxis } from '@elastic/eui/lib/experimental';
+import { AreaChartData, WrappedByAutoSizer, ChartHolder } from '.';
+import { AutoSizer } from '../auto_sizer';
+
+export const ChartBaseComponent = pure<{
+  data: AreaChartData[];
+  width: number | undefined;
+  height: number | undefined;
+}>(({ data, ...chartConfigs }) =>
+  chartConfigs.width && chartConfigs.height ? (
+    // @ts-ignore
+    <SeriesChart {...chartConfigs} showDefaultAxis={false} xType="ordinal">
+      {data.map(series =>
+        series.value != null ? (
+          /**
+           * Placing ts-ignore here for fillOpacity
+           * */
+          // @ts-ignore
+          <EuiAreaSeries
+            key={`stat-items-areachart-${series.key}`}
+            name={series.key.replace('Histogram', '')}
+            // @ts-ignore
+            data={series.value}
+            fillOpacity={0.04}
+            color={series.color}
+          />
+        ) : null
+      )}
+      {/*
+// @ts-ignore */}
+      <EuiXAxis tickFormat={timestamp => timestamp.split('T')[0]} />
+      {/*
+// @ts-ignore */}
+      <EuiYAxis />
+    </SeriesChart>
+  ) : null
+);
+
+export const AreaChart = pure<{ areaChart: AreaChartData[] | [] | null | undefined }>(
+  ({ areaChart }) => (
+    <AutoSizer detectAnyWindowResize={false} content>
```

Comment:
Apply `AutoSizer` to fix it crashes after screen re-sizing, and align `ChartHolder ` vertical center, but I couldn't test AreaChart properly in this case, as no height/width available. So exporting every component here to verify them, please let me know if there's better way to do this. 
<img width="1661" alt="Screenshot 2019-05-06 at 13 13 16" src="https://user-images.githubusercontent.com/6295984/57207637-85962a80-7001-11e9-99e0-2d855eda6eb3.png">
<img width="1660" alt="Screenshot 2019-05-06 at 13 13 37" src="https://user-images.githubusercontent.com/6295984/57207638-862ec100-7001-11e9-9524-880536f85d11.png">